src/scanoss/cli.py

```
@@ -1390,6 +1391,29 @@ def comp_versions(parser, args):
+    Run the "component semgrep" sub-command
+    Parameters
+    ----------
+        parser: ArgumentParser
+            command line parser object
+        args: Namespace
+            Parsed arguments
+    """
+    if (not args.purl and not args.input) or (args.purl and args.input):
+        print_stderr('Please specify an input file or purl to decorate (--purl or --input)')
+        parser.parse_args([args.subparser, args.subparsercmd, '-h'])
+        exit(1)
```

Ruff (0.8.2)

Use `sys.exit()` instead of `exit`

Replace `exit` with `sys.exit()`

(PLR1722)